inc/class-emitter.php

```diff
@@ -41,6 +41,13 @@ class Emitter {
 	const HEADER_KEY = 'Surrogate-Key';
 
 	/**
+	 * Maximum header length.
+	 *
+	 * @var integer
+	 */
+	const HEADER_MAX_LENGTH = 4096;
```

Comment:
Why did you choose this size? The real-world constraint is something close to 32kb